application/src/test/java/org/togetherjava/tjbot/features/basic/SlashCommandEducatorTest.java

```diff
@@ -60,11 +60,25 @@ void ignoresOtherMessages(String message) {
         verify(event.getMessage(), never()).replyEmbeds(any(MessageEmbed.class));
     }
 
+    @ParameterizedTest
+    @MethodSource("provideOtherMessages")
+    void ignoresMessagesWhereLengthIsGreaterThanThirty(String message) {
+        // GIVEN a message's length is more than Thirty
+        // WHEN the message is sent
+        MessageReceivedEvent event = sendMessage(message);
+
+        // THEN the system ignores the message and does not reply to it
+        verify(event.getMessage(), never()).replyEmbeds(any(MessageEmbed.class));
+    }
```

Comment:
this test is unecessary. the new messages are already covered by the existing `void ignoresOtherMessages(String message) {` method right above this one :)

Reply:
okat, removing it.